docs/guides/external-node/00_quick_start.md

```diff
@@ -65,12 +65,12 @@ The HTTP JSON-RPC API can be accessed on port `3060` and WebSocket API can be ac
 
 > [!NOTE]
 >
-> To stop state growth, you can enable state pruning by uncommenting `EN_PRUNING_ENABLED: true` in docker compose file,
+> To stop historical DB growth, you can enable DB pruning by uncommenting `EN_PRUNING_ENABLED: true` in docker compose file,
```

Comment:
@slowli we have merkle tree pruning too, don't we? Or wasn't this project completed?

Reply:
We do. But I think we are only pruning historical tree states.